### ISSUE TITLE
I don't understand the lamp icon

### ISSUE THREAD

**Issue by honzajavorek** [twbs/icons#916]
Hello, the icon set is an awesome feature of the new version of Bootstrap! I appreciate it a lot. But I've noticed this icon
in the list and I couldn't say what it is.

<img width="300" alt="Screenshot 2021-06-15 at 23 29 27" src="https://user-images.githubusercontent.com/283441/122126104-9137ba80-ce31-11eb-8fbc-baded9e1fbbc.png">

Even when I read that it's a "lamp", I still don't get the icon. I checked with some other people and nobody could say what it is. I don't know if this qualifies as a bug, feel free to close this :)

https://icons.getbootstrap.com/icons/lamp/

**Comment by RubenVerg** [comment]
It's a hanging lamp from the ceiling, the drop shape is the lightbulb and the rectangle is I suppose glass or anyways something to hold the lamp

- RubenVerg ❤️


________________________________
From: Honza Javorek ***@***.***>
Sent: Tuesday, June 15, 2021 11:32:19 PM
To: twbs/icons ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [twbs/icons] I don't understand the lamp icon (#916)


Hello, the icon set is an awesome feature of the new version of Bootstrap! I appreciate it a lot. But I've noticed this icon
in the list and I couldn't say what it is.

[Screenshot 2021-06-15 at 23 29 27]<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F283441%2F122126104-9137ba80-ce31-11eb-8fbc-baded9e1fbbc.png&data=04%7C01%7C%7C5ad2f869f9194c52b51e08d930450e9a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637593895411958374%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=avktxb%2Fr5rcTcd0iJCwLgvCPukDur53a1TxAuqkzLXs%3D&reserved=0>

Even when I read that it's a "lamp", I still don't get the icon. I checked with some other people and nobody could say what it is. I don't know if this qualifies as a bug, feel free to close this :)

https://icons.getbootstrap.com/icons/lamp/<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Ficons.getbootstrap.com%2Ficons%2Flamp%2F&data=04%7C01%7C%7C5ad2f869f9194c52b51e08d930450e9a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637593895411968369%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=uBpbvcZQ7%2BSFqohjkQ0xh%2FKt5wN14QltvCdpKCWm7fo%3D&reserved=0>

**Comment by vanillajonathan** [comment]
I agree, it is quite difficult to grasp what it means. It is not immediately obvious, hence it is not "iconic".

**Comment by honzajavorek** [comment]
Oh, thanks for the explanation. The issue then is, icons probably shouldn't need explanation 😅

**Comment by RubenVerg** [comment]
I mean I almost instantly realized what it was but only because I knew it was a lamp (and because I basically spend half an hour a day browsing these icons), maybe it would've been difficult to figure out without caption

**Comment by mdo** [comment]
Yeah, it's trash lol. I'll redraw it in an upcoming release :).

**Comment by deandob** [comment]
Actually my first impressions was that it was looking down on a toilet! Lol

**Comment by Maelstorm1973** [comment]
Actually, instead of throwing it away, rename it as toilet.  May have to redraw it a little bit, but it would be interesting.  How many icon sets have you seen that includes a toilet?

**Comment by RubenVerg** [comment]
not a fan, especially of the beady things down the side. can we keep that (or the original one I suppose) as `lamp-alt`?

**Comment by honzajavorek** [comment]
We might anger the folks who use the current icon as toilet, but for the lamp crowd, I think this is great improvement ❤️
